Placeholder "@libdir@" will be substituted by "${exec_prefix}/lib" after
caja-python.pc.in transformed into caja-python.pc, but ${exec_prefix} isn't
defined here, which leads to an error when parsing the pc file.

--- a/caja-python.pc.in
+++ b/caja-python.pc.in
@@ -3,6 +3,7 @@
 Version: @VERSION@
 
 prefix=@prefix@
+exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
 libdir=@libdir@
 pythondir=@datadir@/caja-python/extensions
